### PR TITLE
Showing pull request flow

### LINKED ISSUE #19: Add bio for carl-potts
Here is my **bio** for the instructor @githubteacher

### PR DESCRIPTION
**Pull request by carl-potts**
This fixes #19 
- [ ] this is an incomplete item

### CODE REVIEW

**Comment by carl-potts** [comment]
For your review @IbraheemYSaleh

File: carl-potts-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+##Bio
```
Comment (carl-potts) [inline carl-potts-bio.md]: Huh?

**Comment by IbraheemYSaleh** [comment]
Is that the location of your secret treasure? :moneybag: 

I also recommend that you complete that very important item! 

What do you think @jveregge

**Comment by jveregge** [comment]
comment, comment and furthermore comment

**Pull request merged**